### DISCUSSION TITLE
Add slash command only for guilds or only for private chat

### DISCUSSION THREAD

**Discussion by Mawobi** [discord-jda/JDA#2083] [Questions and Help]
Hello,

I want to create a command that is only available in guild channels and not in private chat with the bot.
I don't want the syntax of the command to be displayed when I type a slash in private chat with the bot.
How is this possible?

With kind regards
Mawobi

**Comment by Tais993** [comment]
Instead of adding your command to JDA, use a Guild.
So Guild#updateCommands iirc

**Reply by Mawobi** [reply]
Thanks! Can I add the command to guilds in general (to all guilds where to bot is added) and not only to one specific guild?

**Reply by Tais993** [reply]
You gotta loop through all guilds then, JDA#getGuildCache

**Reply by Mawobi** [reply]
Alright, thanks much!
